docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -1,35 +1,26 @@
 # How To Upgrade Legacy Map Features
 
-A list of (case sensitive) changes to upgrade legacy TripleA Game XML properties and options
-to the most recent.
+A list of changes to upgrade legacy TripleA Game XML properties and options to the most recent.
  
-When changes are required, all occurrences of the code before the `>>` must be substituted
-with the code after the `>>`. In any code, the "\*" symbol means whatever (the change or deletion is
-to be applied to every given code having whatever character or characters in place of the "\*").
-
-## Upgrade from 1.8.0.9 to 1.9.0.0.3266
-
-Note: The following list covers all basic upgrade cases for maps, but it may not upgrade "Property"
-options (playerProperty, unitProperty...) of ineffective items you must delete (such changes have
-been omitted from this document for brevity).
```

Comment:
This should have not been removed. Since the document is not informing how to upgrade such items, I think it is good having a mention telling that such items are not covered by the information given. For example, if one has a map with a trigger changing the value of "giveUnitControl" from false to true, that item will not be touched, by following the information at this documentation literally (it would be needed manually to delete the whole trigger, since the option is useless both at the "true" and at the "false" values, thus the trigger is useless too). I decided not to provided such information as the case is remote, and merely noticing that "Property" options are not touched (to avoid confusion, I'll point out that property options are not properties (the term "property" in TripleA means at least three different and largely unrelated things, as far as I know)).

Reply:
Hmm, the sentence needs work.  For example, the following sentence does not make sense:   "but it may not upgrade "Property"options " . 

We do have a section of "delete" these items. Perhaps we're only saying that this document is not complete? Beyond that, practically speaking, I'm not sure how this distinction is actually important to someone upgrading their map from 1.8 to later. Are you saying this document is, by design, not even complete for that?

Reply:
Yes, that is what I'm saying and what that part is saying. I think it's clear enough that it is incomplete: "it may not upgrade "Property" options (playerProperty, unitProperty...) of ineffective items you must delete".

Reply:
That section tells to delete the options in the attachments, but it is not necessarily good enough if you happen to have the same options referenced in triggers (that is having triggers that change the options). Since this is a remote possibility, and it is particularly complex to tell how to handle it (it may or may not be the only thing that the trigger is doing), especially for a deletion case, I've left it out.